pkg/controller/bastion/actuator_delete.go

```diff
@@ -32,36 +32,36 @@ func (a *actuator) Delete(ctx context.Context, bastion *extensionsv1alpha1.Basti
 
 	opt, err := DetermineOptions(ctx, bastion, cluster)
 	if err != nil {
-		return errors.Wrap(err, "failed to setup GCP options")
+		return fmt.Errorf("%w, failed to setup GCP options", err)
 	}
 
 	gcpClient, err := a.getGCPClient(ctx, bastion)
 	if err != nil {
-		return errors.Wrap(err, "failed to create GCP client")
+		return fmt.Errorf("%w, failed to create GCP client", err)
 	}
 
 	if err := removeFirewallRule(ctx, logger, bastion, gcpClient, opt); err != nil {
-		return errors.Wrap(err, "failed to remove firewall rule")
+		return fmt.Errorf("%w, failed to remove firewall rule", err)
 	}
 
 	if err := removeBastionInstance(ctx, logger, gcpClient, opt); err != nil {
-		return errors.Wrap(err, "failed to remove bastion instance")
+		return fmt.Errorf("%w, failed to remove bastion instance", err)
 	}
 
 	deleted, err := isInstanceDeleted(ctx, gcpClient, opt)
 	if err != nil {
-		return errors.Wrap(err, "failed to check for bastion instance")
+		return fmt.Errorf("%w, failed to check for bastion instance", err)
 	}
 
 	if !deleted {
 		return &ctrlerror.RequeueAfterError{
 			RequeueAfter: 10 * time.Second,
-			Cause:        errors.New("bastion instance is still deleting"),
+			Cause:        fmt.Errorf("bastion instance is still deleting"),
```

Comment:
nit: in this case you can use errors.New from the stdlib

Reply:
committed in 207331e931675da727e95398fd32351b78dd480c

btw, from my local testing ` fmt.Errorf` seems also working as expected same as `errors.New`.